decidim-proposals/db/migrate/20240110203500_add_withdrawn_fields_on_proposals.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+class AddWithdrawnFieldsOnProposals < ActiveRecord::Migration[6.1]
```

Comment:
```suggestion
class AddWithdrawnAtFieldToProposals < ActiveRecord::Migration[6.1]
```

Reply:
I will manually push this, as the file needs to be renamed (to avoid confusing zeitwerk)

Reply:
Done at [910f474](https://github.com/decidim/decidim/pull/12289/commits/910f474d8d70bac7973a88520e3ac8ae19ea7210)